### DISCUSSION TITLE
TrackHat new head tracking device testing

### DISCUSSION THREAD

**Discussion by sthalik** [opentrack/opentrack#1364] [General]
Hey,

The company [TrackHat](https://www.trackhat.org/) is planning to release a new head tracking device. It's a sensor that detects LED point centers in hardware, similar to the Wii and PT trackers. It has its own strengths and weaknesses compared to using the PS3 Eye.

You can test it if,

- you play fast-paced sims like DCS, Falcon BMS, or Arma
- know how to set up mappings, PT, and the Accela filter
- want to do multiple rounds of testing with in-between instructions and feedback
- want to test it for an excess of a week

You get to receive a camera circuit board (no case or mounting handle, yet) with a USB connector, and a battery-powered IR clip. Following the end of testing, you'll be able to return the device in exchange for a finished product with a mounting handle and possibly minor hardware/firmware changes. The testing operates entirely on an informal basis, with no contracts, NDAs, or penalties.

Due to silicon shortage, there are very few available units for testing at this time. We're missing STM32 controllers for the sensor's board before more widespread testing. I have a shortlist of people who receive priority if they volunteer for testing. However, only volunteer if you're interested in the device, NOT as a service to the opentrack project!

The shortlist (in random order):

- @Len62
- @MathijsG
- @DaWelter 
- @Slion 
- @GO63-samara 
- @orokhovatskiy 
- @cpuwolf 

~~At this point there are up to 3 units available for testing. Please respond here or through email.~~

**Update: The testers have already been selected.**

-sh

**Comment by orokhovatskiy** [comment]
Great news, thanks for announce!

I don’t have Windows PC for testing now. So I couldn’t help in testing now.
__
Best regards,
Alexander Orokhovatskiy | +7(930)719-71-14

From: Stanislaw Halik ***@***.***>
Reply to: opentrack/opentrack ***@***.***>
Date: Thursday, 4 November 2021, 20:09
To: opentrack/opentrack ***@***.***>
Cc: Alexander Orokhovatskiy ***@***.***>, Mention ***@***.***>
Subject: [opentrack/opentrack] TrackHat new head tracking device testing (Discussion #1364)


Hey,

The company TrackHat<https://www.trackhat.org/> is planning to release a new head tracking device. It's a sensor that detects LED point centers in hardware, similar to the Wii and PT trackers. It has its own strengths and weaknesses compared to using the PS3 Eye.

You can test it if,

  *   you play fast-paced sims like DCS, Falcon BMS, or Arma
  *   know how to set up mappings, PT, and the Accela filter
  *   want to do multiple rounds of testing with in-between instructions and feedback
  *   want to test it for an excess of a week

You get to receive a camera circuit board (no case or mounting handle, yet) with a USB connector, and a battery-powered IR clip. Following the end of testing, you'll be able to return the device in exchange for a finished product with a mounting handle and possibly minor hardware/firmware changes. The testing operates entirely on an informal basis, with no contracts, NDAs, or penalties.

Due to silicon shortage, there are very few available units for testing at this time. We're missing STM32 controllers for the sensor's board before more widespread testing. I have a shortlist of people who receive priority if they volunteer for testing. However, only volunteer if you're interested in the device, NOT as a service to the opentrack project!

The shortlist (in random order):

  *   @Len62<https://github.com/Len62>
  *   @MathijsG<https://github.com/MathijsG>
  *   @DaWelter<https://github.com/DaWelter>
  *   @Slion<https://github.com/Slion>
  *   @GO63-samara<https://github.com/GO63-samara>
  *   @orokhovatskiy<https://github.com/orokhovatskiy>
  *   @cpuwolf<https://github.com/cpuwolf>

At this point there are up to 3 units available for testing. Please respond here or through email.

-sh

**Comment by Len62** [comment]
@sthalik 
Since it appears no one signed-up yet I would be happy to test this for you.

**Comment by sthalik** [comment]
~~That's great! Come join <<https://discord.gg/NBVX6arwnv>>.~~

**Comment by MathijsG** [comment]
I just stumbled upon this one. Is there still a test slot available? :)

**Comment by Len62** [comment]
I don't think so, I got notified via the discord link that they were filled.

**Comment by sthalik** [comment]
@Len62 have you been able to volunteer to James?

**Comment by Len62** [comment]
Yes, here is the text from discord:
James Allen
11/10/2021
Hi there
welcome to the testing discord 🙂
have you read the email thoroughly?

Gramps (my discord name)
 11/10/2021
I have not yet received an email.

James Allen
11/10/2021
Hi, thank you for applying, the testing positions have been filled!

I should have volunteered right away when I got the notice from you. Thanks for thinking of me.

**Comment by 5YNTH3T1K** [comment]
Hey is this still going? I use ARMA, rise of flight and Il-2 .

**Comment by sthalik** [comment]
No, sorry. The new sensor and board have a very limited number of units at this time. There's gonna be a larger test with 1k units being made. Right now it's all paused due to the silicon shortage. So the test is going on with less than 10 people.

**Reply by MexxPex** [reply]
Do you have any idea of how people would find out about the larger test? 

I would like to partake in any test whenever it happens as I do play sims like Arma 3.
